### ISSUE TITLE
"All submitted files" download does not correctly name questions with subfolders

### ISSUE THREAD

**Issue by nicknytko** [PrairieLearn/PrairieLearn#4096]
Example: https://www.prairielearn.org/pl/course_instance/128491/instructor/assessment/2310701/downloads/CS_357_Spring2021_Q6_all_files.zip

For question `16-SVD/SVD-LowRank-Image-Denoising`, folders are truncated as `netid_1_16-SVD` instead of having the full question id.

**Comment by nicknytko** [comment]
Actually, what is this file supposed to look like?  One big zip with all submitted files in the root?

**Comment by mwest1066** [comment]
Yep, should include all submitted files in the zip, inside one flat directory.

**Comment by nicknytko** [comment]
I guess this isn't failing explicitly then, but somewhere the `/` in the question names are getting interpreted as folders (probably by the zip library)

**Comment by mwest1066** [comment]
Good catch. When we added subfolders for questions we didn't think about the fact that QIDs will now contain `/` and this will mess up filename creation.

What's the right fix? Replace `/` with `_` for both the zip file name and all student submitted filenames?

**Comment by nicknytko** [comment]
Yep, I think that'd be the easiest fix here.

**Comment by jonatanschroeder** [comment]
Closing in favour of #7715.
